torch/_inductor/kernel/flex_decoding.py

```diff
@@ -46,12 +46,12 @@ def flex_decoding_grid(batch_size, num_heads, n_keys, d_model, meta):
     # BLOCK_N: block size of K & V along N dimension.
     #
     # change of base out of the loop
-    # ROWS_GUARANTEED_SAFE: Is it guaranteed that at least one value in each row
+    # rows_guaranteed_safe: Is it guaranteed that at least one value in each row
```

Comment:
I would prefer for these to be kept all uppercase - it differentiates between kernel params and local params.

Reply:
Should we differentiate input pointers and constant parameters?

Reply:
Personally I think it's mostly not that hard to figure out.